recipes/enchant/all/conanfile.py

```diff
@@ -0,0 +1,52 @@
+import functools
+
+from conans import ConanFile, CMake, tools
+
+required_conan_version = ">=1.43.0"
+
+
+class EnchantConan(ConanFile):
+    name = "enchant"
+    description = (
+        "Enchant aims to provide a simple but comprehensive abstraction for "
+        "dealing with different spell checking libraries in a consistent way"
+    )
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://abiword.github.io/enchant/"
+    topics = "enchant", "spell", "spell-check"
+    license = "LGPL-2.1-or-later"
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "cmake", "cmake_find_package_multi"
+    requires = "glib/2.71.3", "hunspell/1.7.0"
+    exports_sources = \
+        "CMakeLists.txt", "configure.cmake", "configmake.h", "patches/*"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def source(self):
+        root = self._source_subfolder
+        get_args = self.conan_data["sources"][self.version]
+        tools.get(**get_args, destination=root, strip_root=True)
```

Comment:
```suggestion
        tools.get(**self.conan_data["sources"][self.version],
                  destination=self._source_subfolder, strip_root=True)
```

Reply:
Black formats this differently. I don't feel this needs change.